examples/kebechet/tooling.md

```diff
@@ -0,0 +1,56 @@
+
+From the architecture schema of [Kebechet](https://github.com/operate-first/data-driven-development/tree/main/examples/kebechet/README.md), you can see that Kebechet requires multiple components and workflows to provide services. The major components and their functions are described below.
+
+## Technical Infrastructure
+
+This section describes the categories of components involved in the operational life of Kebechet application:
+
+- `service` -> services that are used by the application and provided by the platform
+
+- `application` -> components or libraries that are created for the application
+
+- `monitoring` -> components or libraries that are created/used for monitoring and pushing metrics
+
+- `analytics` -> components or libraries that are created/used for providing analysis on the data collected
+
+
+### Service components
+
+* [Kafka](https://kafka.apache.org/) - an open-source distributed event streaming platform.
+* [Argo](https://argoproj.github.io/) - family of open source tools for Kubernetes to run workflows, manage clusters, and do GitOps right.
+* [Ceph](https://ceph.io/en/) - an open-source, distributed storage system.
+
+### Application components
+
+* [PostgreSQL Database](https://www.postgresql.org/) - open source object-relational database.
+* [Thoth storages](https://github.com/thoth-station/storages) - Thoth storages and database adapter.
+* [Thoth User API](https://github.com/thoth-station/user-api) - Thoth service interface and Kafka producer.
+* [Thoth Investigator](https://github.com/thoth-station/investigator) - Kafka consumer of messages sent by different components.
+* [Argo workflow controller](https://argoproj.github.io/argo-workflows/workflow-controller-configmap/) - Controller for scheduling and handling Argo workflows.
+* [Argo workflows](https://argoproj.github.io/argo-workflows/). Each of them contain several steps:
+
+    - [Kebechet Administrator Argo workflow](https://github.com/thoth-station/thoth-application/blob/master/kebechet/base/openshift-templates/kebechet-administrator.yaml) - run Kebechet administrator that decides based on triggers which repo needs to receive a Kebechet update.
+    - [Adviser Argo workflow](https://github.com/thoth-station/thoth-application/blob/master/adviser/base/openshift-templates/adviser.yaml) - run Thoth adviser recommender system.
+    - [Kebechet Argo workflow](https://github.com/thoth-station/thoth-application/blob/master/kebechet/base/openshift-templates/kebechet.yaml) - run Kebechet component logic.
+    - [Solver Argo workflow](https://github.com/thoth-station/thoth-application/blob/master/solver/base/openshift-templates/solve_and_sync-template.yaml) - run solver component to analyze python packages for different runtime environments, stores knowledge in the graph database and sends messages for scheduling Security workflows.
+
+
+### Monitoring Components
+
+* [Prometheus](https://prometheus.io/) - Monitoring tool for collecting time series metrics.
```

Comment:
Can you also please add "To learn more about Prometheus, refer to this documentation [here](https://github.com/AICoE/data-driven-development/blob/main/data-sources/prometheus.md)."

Reply:
Nice! yes :)